package.json

```diff
@@ -32,7 +32,7 @@
     "babel-preset-stage-2": "6.24.1",
     "gh-pages": "1.2.0",
     "node-sass": "4.9.0",
-    "parcel-bundler": "1.9.4",
+    "parcel-bundler": "parcel-bundler/parcel",
```

Comment:
I'd like to wait for the changes to be part of a version release. We'll need to update the version here once that's been done.